packages/cra-template-mephisto-review/template/src/components/WordCloud.jsx

```diff
@@ -198,7 +198,7 @@ function WordCloud({
 
   //map css styles to spans with corresponding word and populate paragraph with spans
   return (
-    <div className="cloud">
+    <div className="word-cloud">
```

Comment:
Shouldn't this WordCloud component be in the plugins folder?

Reply:
This part just renders the div full of spans for the actual word cloud itself (can be resized inside other divs), it plugs into two wrapper components for both the word cloud for the ItemView and the AllItemView

Reply:
Could you point me to the two wrapper components?

I was thinking that all wordcloud stuff would go into the plugins folder (preferrably in it's own folder). that way if someone else in the future wants to add a new plugin, they could just follow the precedent and drop a folder into the plugins directory and use it. if we have the files scattered, it makes the plugins less modular and plug-and-play.

we can discuss this architecture further. otherwise this PR looks pretty awesome.

Reply:
ideally, if the code is all in one place, it's more packageable, say in the future if we decide to move it into an npm package. e.g: `npm install mephisto-plugin-wordcloud` or something.

Reply:
I totally agree, I was testing moving the css, etc. into their own folders for simplicity, but decided to hold off to keep the PR smaller. I think I will restructure the folders for this PR to align to this idea after we clear up the CSS issues.

Reply:
The wrapper components are in /plugin/WordCloudItemRenderer and /plugins/WordCloudItemViewRenderer